### PR TITLE
clear output dir

### PR DESCRIPTION
**Pull request by itaysk**
add flag to clear the output dir before running
(I think output dir is better name than output path)

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -39,7 +39,7 @@ func main() {
 				PerfBufferSize:        c.Int("perf-buffer-size"),
 				PidsToTrace:           c.IntSlice("pid"),
 				BlobPerfBufferSize:    c.Int("blob-perf-buffer-size"),
-				OutputPath:            c.String("output-path"),
+				OutputDir:             c.String("output-dir"),
```
Comment (yanivagman) [inline main.go:42]: Why changing path to dir?
Today we save to a dir, maybe tomorrow to tar.gz, who knows?

File: main.go [hunk 2]
```diff
@@ -135,14 +138,20 @@ func main() {
...
 			&cli.StringFlag{
-				Name:  "output-path",
+				Name:  "output-dir",
 				Value: "/tmp/tracee",
-				Usage: "set output path",
+				Usage: "set output dir",
+			},
+			&cli.BoolFlag{
+				Name:    "clear-output-dir",
+				Aliases: []string{"rm"},
```
Comment (yanivagman) [inline main.go:147]: Using rm as a shortcut may be a bit confusing, don't you think?

File: main.go (hunk 1 above)
Comment (itaysk) [reply main.go]: ok

File: main.go (hunk 2 above)
Comment (itaysk) [reply main.go]: I changed it to clear, is it better?

**Review by yanivagman — APPROVED** [review]

**Pull request merged**